mkdocs/docs/how-to-release.md

```diff
@@ -128,21 +144,51 @@ svn add $SVN_TMP_DIR_VERSIONED
 svn ci -m "PyIceberg ${VERSION}" ${SVN_TMP_DIR_VERSIONED}
 ```
 
-### Upload to PyPi
+Verify the artifact is uploaded to [https://dist.apache.org/repos/dist/dev/iceberg](https://dist.apache.org/repos/dist/dev/iceberg/).
```

Comment:
Can we add a section for deleting old RCs?

Reply:
```
svn delete https://dist.apache.org/repos/dist/dev/iceberg/pyiceberg-<OLD_RC_VERSION> -m "Remove old RC artifacts"
```

I think this is the right command, will test during 0.8.1 release